### DISCUSSION TITLE
Issue with GitHub Student Developer Pack Verification

### DISCUSSION THREAD

**Discussion by tusharthite15** [community/community#118727] [GitHub Education]
### Hi everyone,

Hello everyone,

I hope this message finds you well.

I am reaching out to the community regarding an issue I've been experiencing with my GitHub Student Developer Pack verification process. I applied for the GitHub Student Developer Pack, and while the usual verification timeframe is between 3 to 5 days, it's been over 45 days now, and my application still hasn't been verified.

Interestingly, when I visit the GitHub Education website, it indicates that my application has been approved and displays the message: "Hi, tusharthite15! You were last verified as a student on Mar 1, 2024. You don't need to reverify at this time. There may be a wait period between verification and access to academic benefits."

However, despite this message, I haven't received access to the benefits associated with the GitHub Student Developer Pack. I've been eagerly waiting to access these benefits to support my learning and development journey.

I've reached out to GitHub support multiple times regarding this issue, but unfortunately, I haven't received a satisfactory response or resolution yet.

I'm reaching out to the community to seek guidance and support on what steps I can take to resolve this issue and gain access to the benefits of the GitHub Student Developer Pack.

If anyone has faced a similar situation or has any advice on how to proceed, I would greatly appreciate your insights and assistance.

Thank you for your time and support.

Best regards,
Tushar Thite

**Comment by samus-aran** [comment]
Hi @tusharthite15 

We can see you have been approved and received academic benefits including the PRO status. We are also unable to find any support tickets raised through your username - have you submitted as another account?

To redeem your academic benefits navigate to the [benefits page](https://education.github.com/pack) and each offer will tell you nave to redeem the offer.

**Comment by rsr077** [comment]
Hi Everyone,
I'm start c language learning . 
60 days on GitHub anyone interested.
